### PR TITLE
WIP - Upgrade Angular to V12

### PR DESCRIPTION
**Pull request by DivergentEuropeans**
## Proposed changes

This PR addresses Issue: [*Link to Github issue within https://github.com/zowe/zlux/issues* if any]

This PR depends upon the following PRs:

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Change in a documentation
- [ ] Refactor the code 
- [ ] Chore, repository cleanup, updates the dependencies.
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## PR Checklist
Please delete options that are not relevant.
- [ ] If the changes in this PR are meant for the next release / mainline, this PR targets the "staging" branch.
- [ ] My code follows the style guidelines of this project (see: [Contributing guideline](https://github.com/zowe/zlux/blob/master/CONTRIBUTING.md))
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes
- [ ] video or image is included if visual changes are made
- [ ] Relevant update to CHANGELOG.md
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works, or describe a test method below

## Testing

Known Issues: Personalization menu is visually buggy, Notifications do not work, Angular Material snackbars are disabled

### CODE REVIEW

**Comment by DivergentEuropeans** [comment]
Code is not ready for review yet

**Comment by lchudinov** [comment]
@DivergentEuropeans please revert the changes for `fetch` back. I double checked that `response.json()` is needed.

File: bootstrap/src/bootstrap/dsm-resources.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ export var bootstrapLogger : ZLUX.ComponentLogger = logger.makeComponentLogger("
 
 fetch('/ZLUX/plugins/org.zowe.zlux.bootstrap/web/assets/i18n/log/messages_en.json')
   .then((response) => {
-    return response.json();
+    return response;
```
[the remark this answers is not in the document]
Comment (lchudinov) [reply bootstrap/src/bootstrap/dsm-resources.ts:31]: Please revert the change back.

File: bootstrap/src/bootstrap/rocket-mvd-resources.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ let environment = new Environment();
 
 fetch('/ZLUX/plugins/org.zowe.zlux.bootstrap/web/assets/i18n/log/messages_en.json')
   .then((response) => {
-    return response.json();
+    return response;
```
[the remark this answers is not in the document]
Comment (lchudinov) [reply bootstrap/src/bootstrap/rocket-mvd-resources.ts:35]: Please revert the change back.

File: system-apps/web-browser-app/webClient/package.json [hunk 3]
```diff
@@ -11,17 +11,17 @@
   "private": true,
   "devDependencies": {
     "@angular-devkit/build-angular": "~0.1102.11",
-    "@angular/animations": "~6.0.9",
+    "@angular/animations": "~12.0.0",
     "@angular/cli": "~8.0.0",
```
Comment (lchudinov) [inline system-apps/web-browser-app/webClient/package.json:15]: Why we don't upgrade `angular/cli` ?

**Review by lchudinov — CHANGES_REQUESTED** [review]
@DivergentEuropeans please revert the changes for `fetch()` back. I double checked that `response.json()` is needed.

**Review by lchudinov — APPROVED** [review]

**Review by DivergentEuropeans — COMMENTED** [review]
This is not ready yet because snapshots and multiple app opens don't work

**Pull request merged**